natterstefan/dictcc
# feat: added targetTranslation audio url support

PR raised in order to add this functionality into the raycast extension :)  

**Play audio with the pronunciation**

https://user-images.githubusercontent.com/27580836/217369631-6da14336-38ba-43e1-b231-367efab9c3d5.mov



# TASK

## What I did
Add to the translations object the property `targetTranslationAudioUrl`

## How to test
- Go to dict.cc, play audio, intercept the requests and check if the url is the same
